Sources/PIALibrary/Account/Data/ClientErrorMapper.swift

```diff
@@ -0,0 +1,64 @@
+
+
+import Foundation
+
+enum UseCaseType {
+    case login
+    case logout
+}
+
+/// Maps an Network Error with a ClientError
+/// The idea is to use this mapper on `PIAWebServices` to map the errors returned from the Swift implementation of the Accounts Lib with the ones that the app expects
+struct ClientErrorMapper {
+    static func map(networkRequestError: NetworkRequestError, useCase: UseCaseType) -> ClientError {
+        switch (networkRequestError, useCase) {
```

Comment:
Just of curiosity, why do we need to know the use case when converting the errors? I'm afraid the switch will get huge if we end up adding all the use cases we need to implement.